lib/ansible/modules/network/ios/_ios_template.py

```diff
@@ -147,9 +147,7 @@ def main():
     candidate = NetworkConfig(contents=module.params['src'], indent=1)
 
     result = {'changed': False}
-    warnings = list()
-    check_args(module, warnings)
-    result['warnings'] = warnings
+    check_args(module)
```

Comment:
@bcoca The reason for me to update the deprecated files is that we don't want people to use those as a reference of how to do it. If we don't have any bad examples, people can't copy them either.